docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -118,6 +118,11 @@ Next, create a Role using this IAM policy to allow Teleport to assume IAM roles:
 <Admonition type="note">
 You can make the policy more strict by providing specific IAM role resource
 ARNs in the Resource field instead of using a wildcard.
+  
+If using the Telport agent deployed in EKS, you cannot use Helm chart
```

Comment:
"Telport" -> "Teleport"

Reply:
Thanks, resolved the spelling error.  So, the interesting part about this note is it is specific to users with Helm chart deployments whereas much of the guide focuses on EC2 usage.  I decided to nestle it under the Step 3 notes since Step 3 applies the IAM permissions to the Teleport agent.  However, I also toyed with the idea of it being under the prerequisites section.  Happy to relocate the section if you feel it should be more visible/appropriate

Reply:
I think "Prerequisites" might be a better home for it. The line, "AWS EC2 or other instance where you can assign a IAM Security Role for the Teleport Agent" indicates that there are various possible environments you can use, and adding the admonition here would make it clear what the limitations are to using a particular environment.

Reply:
updated to add a note admonition at the bottom of prereq section with the same information